network-api/networkapi/highlights/factory.py

```diff
@@ -18,17 +18,12 @@ class Meta:
             "footer_sentence",
         )
 
-    class Params:
```

Comment:
These details seem a bit unnecessary to me. I'm deleting it to avoid complication.

Reply:
Cool I'll add them back!